src/core/js/a11y/popup.js

```diff
@@ -1,225 +1,218 @@
-define([
-  'core/js/adapt'
-], function(Adapt) {
+import Adapt from 'core/js/adapt';
 
-  /**
-   * Tabindex and aria-hidden manager for popups.
-   * @class
-   */
-  var Popup = Backbone.Controller.extend({
+/**
+ * Tabindex and aria-hidden manager for popups.
+ * @class
+ */
+export default class Popup extends Backbone.Controller {
 
+  initialize() {
     /**
      * List of elements which form the base at which elements are generally tabbale
      * and aria-hidden='false'.
      *
      * @type {Array<Object>}
      */
-    _floorStack: [$('body')],
+    this._floorStack = [$('body')];
     /**
      * List of elements to return the focus to once leaving each stack.
      *
      * @type {Array<Object>}
      */
-    _focusStack: [],
+    this._focusStack = [];
     /**
      * Hash of tabindex states for each tabbable element in the popup stack.
      *
      * @type {Object}
      */
-    _tabIndexes: {},
+    this._tabIndexes = {};
     /**
      * Hash of aria-hidden states for each tabbable element in the popup stack.
      *
      * @type {Object}
      */
-    _ariaHiddens: {},
+    this._ariaHiddens = {};
     /**
      * Incremented unique ids for elements belonging to a popup stack with saved
      * states,
      */
-    _elementUIDIndex: 0,
-
-    initialize: function() {
-      this.listenTo(Adapt, {
-        'popup:opened': function($element, ignoreInternalTrigger) {
-          if (ignoreInternalTrigger) {
-            return;
-          }
-          Adapt.a11y.log.deprecated('Adapt.trigger("popup:opened", $element) is replaced with Adapt.a11y.popupOpened($element);');
-          this.opened($element, true);
-        },
-        'popup:closed': function($target, ignoreInternalTrigger) {
-          if (ignoreInternalTrigger) {
-            return;
-          }
-          Adapt.a11y.log.deprecated('Adapt.trigger("popup:closed", $target) is replaced with Adapt.a11y.popupClosed($target);');
-          this.closed($target, true);
+    this._elementUIDIndex = 0;
+    this.listenTo(Adapt, {
+      'popup:opened'($element, ignoreInternalTrigger) {
```

Comment:
Is it okay to set a function like this?

Reply:
Apparently so. No linting, nor parsing nor transpiring errors... So just a question of preference or conflicting standards at this point I think?